ELLI/src/app/features/signup/signup.component.html

```diff
@@ -1,7 +1,7 @@
 <section class="login-container">
     <div class="container">
-        <h2 class="center">Signup</h2>
-        <p class="login-link" (click)="showLogin()">Login</p>
+        <h2 class="center">Login</h2>
```

Comment:
needs to read Sign Up

Reply:
okay will take a note for that and discuss tmmrw.